kickstart/finish.erb

```diff
@@ -2,15 +2,8 @@
 kind: finish
 name: Kickstart default finish
 oses:
```

Comment:
there was no RHEL of any version associated before and then the `redhat_register` snippet should also go into there, I presume?